HPCC-18621 Document Std.Uni.Version()

JamesDeFabia opened the pull request:
Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>
@ghalliday Please review
<!-- Thank you for submitting a pull request to the HPCC project

 PLEASE READ the following before proceeding.

 This project only accepts pull requests related to open JIRA issues.
 If suggesting a new feature or change, please discuss it in a JIRA issue first.
 If fixing a bug, there should be an issue describing it with steps to reproduce.
 The title line of the pull request (and of each commit within it) should refer to the
 associated issue using the format:

 HPCC-nnnnn Short description of issue

 This will allow the Jira ticket to be automatically updated to refer to this pull request,
 and will ensure that the automatically-generated changelog is properly formatted.
 Where a pull request contains a single commit the pull request title will be set automatically,
 assuming that the commit has followed the proper guidelines.

 Please go over all the following points, and put an `x` in all the boxes that apply. You may find
 it easier to press the 'Create' button first then click on the checkboxes to edit the comment.
-->

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:
<!-- Please describe how this change has been tested.-->

<!-- Thank you for taking the time to submit this pull request and to answer all of the above-->

Commit:
HPCC-18621 Document Std.Uni.Version()

Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>

Signed-off-by: Jim DeFabia <jamesdefabia@lexisnexis.com>

ghalliday commented:
@JamesDeFabia the text looks fine, the PR does not build, and no check boxes are completed.

JamesDeFabia commented:
@ghalliday I checked the boxes now and am looking into the build failure. I will @ tag you when it is ready to review

HPCCSmoketest commented:
Automated Smoketest: :x:
OS: centos 7.2.1511 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: 096bc79d6db398b73691346e5bbd18313a437927
Number of Error(s): 113
/mnt/disk1/home/vamosax/smoketest/PR-11238/HPCC-Platform/docs/EN_US/ConfiguringHPCC/ConfiguringHPCC.xml:45: element include: XInclude error : could not load /mnt/disk1/home/vamosax/smoketest/PR-11238/HPCC-Platform/docs/EN_US/common/Version.xml, and no fallback was found

[Build error:](https://gist.github.com/ffe137aae6bc175e77eeb6f95128f9b3)
Build log [Build log](https://gist.github.com/HPCCSmoketest/ffe137aae6bc175e77eeb6f95128f9b3#file-result-18-05-29-09-56-00-log)

JamesDeFabia commented:
@ghalliday I fixed the issue and squashed, so back to you.
NB: I had to turn the ECL code example into an image because the current PDF FO Processor  doesn't support these characters. I created a Jira to fix that, but that is for another day.
